pkgs/misc/qtscrcpy/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libsForQt5
+, copyDesktopItems
+, android-tools
+, makeDesktopItem
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qtscrcpy";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "barry-ran";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-2d7AIUra7Uc/N0Ako8JYo07GUTPNgorxl2hmeDJOGPU=";
+  };
+
+  nativeBuildInputs = [ cmake libsForQt5.wrapQtAppsHook copyDesktopItems ];
+  buildInputs = with libsForQt5.qt5 ;[ qttools.dev qtx11extras.dev ];
+
+  patchPhase = ''
+    sed -i 's@"../../../third_party/adb/linux/adb"@"${android-tools.out}/bin/adb"@g' ./QtScrcpy/main.cpp
+    sed -i 's@"../../../third_party/scrcpy-server"@"'"$out"'/scrcpy-server"@g' ./QtScrcpy/main.cpp
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin/ && mkdir -p $out/share/pixmaps/
+    cp /build/source/output/linux/release/QtScrcpy $out/bin/
```

Comment:
Unhardcode the path, I'm pretty sure build would fail on non-sandboxed Nix installations